woocommerce/woocommerce-ios
# [IPPInAppFeedback] UI banner component design tweaks

Part of https://github.com/woocommerce/woocommerce-ios/issues/8530
Closes: #8674 

## Description
This PR updates the  In-Person Payments Feedback banner UI design to make it closer to the [design proposal](https://www.figma.com/file/T16Ov1hzHWLJSOr8iN1BgR/Feature-feedback?node-id=1%3A142&t=HXtzjijE8IxNLnWx-0).

## Changes
- Adds a new `commentContent` icon
- Overrides the default `.purple` color for the Action links with `.pink` (confirmed with design at C0354HSNUJH/p1674112248357019-slack-C0354HSNUJH)
- Appends a new top separator to the component by calling the existing `createBorderView()` in the main `UIStackView`

@toupper as [you mentioned in the prior PR](https://github.com/woocommerce/woocommerce-ios/pull/8640#issuecomment-1383809981), the banner text doesn't go below the `X` icon for dismissal, but I haven't found a way to make this happen with the current implementation without having to modify it entirely. Happy to hear if you have any pointer or recommendation! 🙇 

## Testing instructions
On a store with COD enabled, and set to US or CA:
1. Navigate to Orders, and see the In-Person Payments Feedback banner 
2. Confirm the icon, and color are correct as per the screenshot below:

| Design proposal | Current design |
|--|--|
| <img width="400" alt="Screenshot 2023-01-23 at 15 30 37" src="https://user-images.githubusercontent.com/3812076/213986346-aaf413eb-c5aa-4fbf-9222-08baee8ea374.png"> | <img width=400 src="https://user-images.githubusercontent.com/3812076/213968591-412ae91f-1d16-4d98-80e3-b49d28200e04.png"> |
